server/src/main/java/org/hyperskill/community/flashcards/card/ExampleDataInitializer.java

```diff
@@ -21,63 +30,84 @@
 @Component
 @Slf4j
 public class ExampleDataInitializer {
+    private static final String exampleCollection = "example";
+    private static final String userCollection = "user";
+    private static final String categoryCollection = "category";
+    private static final String userJsonPath = "/json/users.json";
+    private static final String flashcardsJsonPath = "/json/flashcards.json";
+    private final ObjectMapper objectMapper = new ObjectMapper();
+    private final UserMapper userMapper;
     private final MongoTemplate mongoTemplate;
 
-    public ExampleDataInitializer(@Qualifier("example") MongoTemplate mongoTemplate) {
+    public ExampleDataInitializer(UserMapper userMapper, MongoTemplate mongoTemplate) {
+        this.userMapper = userMapper;
         this.mongoTemplate = mongoTemplate;
     }
 
```

Comment:
not important. - but just in general:
you can nicely save this constructor boiler plate with `@RequiredArgsContructor`

Reply:
Thanks for suggestion, done